Sources/Moya/Plugins/NetworkLoggerOutput.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NetworkLoggerOutput.swift
+//  Moya
+//
+//  Created by Alan Yeo on 24/4/18.
+//
+
+import Foundation
+
+public protocol NetworkLoggerOutput {
+    func print(_ separator: String, terminator: String, items: Any...)
+    func format(_ loggerId: String, date: String, identifier: String, message: String) -> String
+}
+
+class DefaultNetworkLoggerOutput: NetworkLoggerOutput {
```

Comment:
Can't we just make this an `open` class and remove the protocol?
`DefaultNetworkLoggerOutput` -> `NetworkLoggerOutputFormatter`

Reply:
Yeah, I agree.